smu/geometry/smu_molecule.py

```diff
@@ -0,0 +1,172 @@
+"""Class that is responsible for building and assessing proposed
+   bonding patterns."""
+
+import operator
+
+from typing import Callable, Dict, List, Optional, Tuple
+
+import numpy as np
+
+from smu import dataset_pb2
+from smu.parser import smu_utils_lib
+
+
+class MatchingParameters:
```

Comment:
At the time I started this, I thought there would be other optional behaviors. Not sure what there might be yet. If we get more optional things, this will be useful. If not, then yes, it could be a single scalar.